src/api/bridgedWithdraw.js

```diff
@@ -19,12 +19,12 @@ const schema = Joi.object({
 })
 
 const validateArg0 = validateWithJoi(schema)('INVALID_METHOD_ARGUMENT')({
-  context: `bridgedWithdrawal`
+  context: 'bridgedWithdrawal'
 })
 
 const endpoint = '/v1/trading/bridgedWithdrawals'
 
-module.exports = async (dvf, data, authNonce, signature) => {
+module.exports = async (dvf, data, authNonce, signature, additionalPayload = {}) => {
```

Comment:
can we move `additionalPayload` under data? or explicitly set the new properties that are being passed? Since the general pattern is to keep authNonce and signature as the last parameters

Reply:
Done in [ec5ee9d](https://github.com/rhinofi/client-js/pull/459/commits/ec5ee9d107c15e87db770bfda1dd597765000145)